apps/codeforafrica/src/components/ArticlePage/ArticlePage.js

```diff
@@ -37,7 +37,7 @@ function ArticlePage({
         title={title}
         date={date}
         tags={tags}
-        summary={summary}
+        excerpt={summary}
```

Comment:
Shouldn't this be just `excerpt` or `excerpt ={excerpt}`

Reply:
Yes, it **should** be. When the PR was created, the content in `lib/index.js` had it as `summary`. With #168 merged, I should be able to update it to `excerpt` I believe.